src/main/resources/application.properties

```diff
@@ -7,6 +7,9 @@ spring.application.version="@project.version@"
 portal.welfare.tos.privacy = https://github.com/pagopa/idpay-portal-welfare-tos/tree/main/privacy
 portal.welfare.tos.tc = https://github.com/pagopa/idpay-portal-welfare-tos/tree/main/tc
 
+onetrust.privacy-notices.client.url=${ONETRUST_PRIVACYNOTICES_URL:https://trial.onetrust.com/api/privacynotice/v2}
+onetrust.privacy-notices.tos.id=${ONETRUST_TOS_ID:TODO}
```

Comment:
Could you set this ENV vars in the helm chart of this application?

The ONETRUST_TOS_ID should be read from a secret (there will be the TODO to decide the name of the secret, here you could write whatever placeholder you want)

The ONETRUST_PRIVACYNOTICES_URL should be an envSpecific configMap values, which should then be setted in all values-ENV.yml files, with a TODO if not known